src/protocols/secure_channel/CASESession.cpp

```diff
@@ -108,7 +108,7 @@ void CASESession::Clear()
     PairingSession::Clear();
 
     mState = kInitialized;
-    Crypto::ClearSecretData(&mIPK[0], sizeof(mIPK));
+    Crypto::ClearSecretData(mIPK);
```

Comment:
Why aren't we just using `AesCcm128Key` for mIPK?  This does this automatically in its destructor.

Reply:
It's never used for AES-CCM, so AesCcm128Key is the wrong type

Reply:
Sorry, let me restate the question then.  Why don't we have a type that does this itself automatically?

Reply:
Mostly because I did not realize that existed.  I'll switch to that.

Reply:
Done.